### PR TITLE
Plumbing, domain classes

### PR DESCRIPTION
**Pull request by carlschroedl**
Addresses #6

### CODE REVIEW

**Comment by carlschroedl** [comment]
This isn't ready for merging yet. I'm mostly hoping for a sanity check on the build.xml and composer files at this point.

File: build.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
```
Comment (carlschroedl) [inline build.xml]: How is this looking so far?
Do you have any pointers on how to grow this?

File: composer.json [hunk 2]
```diff
@@ -5,7 +5,8 @@
     "require": {
     },
     "require-dev": {
-        "pds/skeleton": "~1.0"
+        "pds/skeleton": "~1.0",
+        "phing/phing": "2.16.0"
```
Comment (carlschroedl) [inline composer.json]: After I added this line for phing, composer complained about being out of sync with the lock file and suggested that I run `composer update`. I did. The diff was big. Is the huge diff for the lock file expected?

File: src/Ballot.php [hunk 3]
```diff
@@ -6,4 +6,11 @@ class Ballot
     /**
      * @todo #4 Implement ballot logic
      */
+	private $listOfListOfCandidates;
+
+  	public function __construct(CandidateListList /*...*/$listsOfCandidates) {
```
Comment (carlschroedl) [inline src/Ballot.php]: I just realized I have php 5.5, so I can't leverage php 5.6+'s varargs syntax yet. I will upgrade soon and uncomment the ellipses.

**Review by andrewshell — CHANGES_REQUESTED** [review]
Looks fine from what I can tell.  Generally, I stick to the [PSR-2](http://www.php-fig.org/psr/psr-2/) coding standards which use 4 spaces instead of tabs, but that's all I noticed. :-)

Just get your PHP upgraded and restore the varargs and it should be fine to merge.

**Comment by carlschroedl** [comment]
Thanks @andrewshell ! Will do. I will add a [phing task](https://akrabat.com/checking-your-code-for-psr-2/) to enforce PSR-2.

**Comment by andrewshell** [comment]
For reference, here is the boilerplate skeleton I typically use for new libraries.  That includes checking for PSR-2: [Cadre.Package](https://github.com/cadrephp/Cadre.Package)

**Comment by carlschroedl** [comment]
Even better! Thanks!

**Comment by andrewshell** [comment]
Any reason not to merge this already? Ideally, pull requests are single features/bug fixes.

**Review by andrewshell — APPROVED** [review]

**Comment by carlschroedl** [comment]
Nope! No reason to wait. I will add a couple more tests in a subsequent PR to get coverage up to 100%. Thanks for reviewing and thanks for merging!

**Comment by andrewshell** [comment]
Also, it will make PRs and merges easier if you do work in feature branches on your fork instead of in your forks 0.x branch.

**Comment by carlschroedl** [comment]
Yeah. I goofed. Sorry. Will do next time!

**Pull request merged**